tests/modules/shasta/test.yml

```diff
@@ -0,0 +1,7 @@
+- name: shasta test_shasta
+  command: nextflow run ./tests/modules/shasta -entry test_shasta -c ./tests/config/nextflow.config -c ./tests/modules/shasta/nextflow.config
+  tags:
+    - shasta
+  files:
```

Comment:
You need to run:
`nf-core modules create-test-yml shasta`

Reply:
When a run this one, the module is killed due to lack of memory. It seems that the create-test-yml execution allows less than 6.GB.

What can I add or modify to increase it for this module?

Reply:
There are files you can use in the `tests/config/test.config` files as test datasets. You should use a small file just to ensure you get output. Doesn't matter if the output makes sense. Just that it runs and makes the output files.